frontend/src/components/MarkdownRenderer/CodeCopyBtn/CodeCopyBtn.tsx

```diff
@@ -0,0 +1,29 @@
+import './CodeCopyBtn.scss';
+
+import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
+import cx from 'classnames';
+import { useState } from 'react';
+
+export default function CodeCopyBtn({
+	children,
+}: {
+	children: any;
+}): JSX.Element {
+	const [isSnippetCopied, setIsSnippetCopied] = useState(false);
+
+	const handleClick = (): void => {
+		navigator.clipboard.writeText(children[0].props.children[0]);
```

Comment:
this will work on the server when server is in https protocol 

https://stackoverflow.com/questions/52054635/copy-clipboard-function-working-locally-but-not-working-server